projects/js-packages/publicize-components/src/components/unified-modal/social-post-preview/content.tsx

```diff
@@ -54,9 +55,17 @@ export function Content( { baseId, selectedConnection, forSmallScreen }: Content
 			id={ `${ baseId }-preview-content-${ selectedConnection.connection_id }` }
 			aria-labelledby={ `${ baseId }-preview-tab-${ selectedConnection.connection_id }` }
 		>
-			<Flex className={ styles.preview } align="center" justify="center">
-				<PostPreview connection={ selectedConnection } />
-			</Flex>
+			{ selectedConnection.enabled ? (
+				<Flex className={ styles.preview } align="center" justify="center">
+					<PostPreview connection={ selectedConnection } />
+				</Flex>
+			) : (
+				<Flex className={ styles[ 'inactive-preview' ] } align="center" justify="center">
+					<p>
+						{ __( 'No post will be shared for this account.', 'jetpack-publicize-components' ) }
```

Comment:
This text may not make much sense for already published/shared posts, as mentioned in the linked issue comment.

Also, if we still want to use it it it IMHO should be


```suggestion
						{ __( 'No post will be shared to this account.', 'jetpack-publicize-components' ) }
```